logstash-core/spec/logstash/api/modules/node_stats_spec.rb

```diff
@@ -69,7 +69,7 @@
       "cpu"=>{
         "total_in_millis"=>Numeric,
         "percent"=>Numeric,
-        "load_average" => { "1m" => Numeric }
+        # load_average is not supported on Windows, set it below
```

Comment:
I confirmed manually that on Elasticsearch (I used ES 5.5.0) only has load_average on Linux. On Windows, the `cpu` object only has `percent`.